runtime/js/fetch.js

```diff
@@ -0,0 +1,66 @@
+import { fetch as fetchImpl } from "ext:deno_fetch/26_fetch.js";
+import { fromInnerResponse, toInnerResponse } from "ext:deno_fetch/23_response.js";
+import { toInnerRequest, fromInnerRequest, Request } from "ext:deno_fetch/23_request.js";
+import { guardFromHeaders } from "ext:deno_fetch/20_headers.js";
+
+let ipfsScheme = "ipfs://";
```

Comment:
```suggestion
const ipfsScheme = "ipfs://";
```

I believe it doesn't need to be mutable

Reply:
Thank you for catching this. I often forget myself and write my JavaScript code in Rust style these days 🙈